infoschema/tables.go

```diff
@@ -1003,6 +1019,68 @@ func dataForTables(ctx sessionctx.Context, schemas []*model.DBInfo) ([][]types.D
 	return rows, nil
 }
 
+func dataForIndexes(ctx sessionctx.Context, schemas []*model.DBInfo) ([][]types.Datum, error) {
```

Comment:
This looks like duplicate code with `fetchShowIndex`? can we extract it to a separate function for reuse?